.github/workflows/ci.yml

```diff
@@ -342,10 +344,17 @@ jobs:
           LICENSE_SERVER: ${{ secrets.LICENSE_SERVER }}
           MAPDL_IMAGE: ${{ env.DOCKER_PACKAGE }}:${{ matrix.mapdl-version }}
 
-      - name: Unit Testing
+      - name: Unit Testing Requirements Installation
         run: |
           pip install -r requirements/requirements_tests.txt
-          xvfb-run pytest -v --maxfail=10 --durations=0 --cov=ansys.mapdl.core --cov-report=xml --cov-report=html
+
+      - name: DPF Server Activation
+        run: |
+          docker run -d -v `pwd`:/dpf -p ${{ env.DPF_PORT }}:50054 ghcr.io/pyansys/dpf-core/dpf:v2021.1
```

Comment:
Can you upgrade this? Ping Paul.

Let's merge without this, it doesn't have to be a blocker.

Reply:
Done. Using latest dev image (``ghcr.io/pyansys/dpf-core:231dev``).

Reply:
I have tried to use the private image, but I cannot make it work. I believe it is an access issue. I have added the PyAnsys bot to DPF repo but still unsuccessful. 
Since DPF standalone is coming at the end of the month, I am going to keep using the public 2021 image. 
Then we can use an installed package and get rid of the docker images.